backend/apps/notifications/models.py

```
@@ -32,7 +32,7 @@ class Notification(models.Model):
 
     class Meta:
         ordering = ["-created_at"]
-        indexes = [models.Index(fields=["recipient", "is_read"])]
+        indexes = [models.Index(fields=["recipient", "is_read"], name="idx_recipientis_read")]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)